SignalServiceKit/src/Storage/TSStorageManager.m

```diff
@@ -198,6 +291,8 @@ - (void)setupDatabaseWithSafeBlockingMigrations:(void (^_Nonnull)())safeBlocking
     [self.database registerExtension:[TSDatabaseSecondaryIndexes registerTimeStampIndex] withName:@"idx"];
     [OWSMessageReceiver syncRegisterDatabaseExtension:self.database];
 
+    setDatabaseInitialized();
```

Comment:
Can you leave a comment as to why we're doing this here after the sync migrations but before the migrations and async registrations?